tests/tck/features/match/MultiLineMultiQueryParts.feature

```diff
@@ -0,0 +1,272 @@
+    When executing query:
+      """
+      USE nba;
+      MATCH (m)-[]-(n) WHERE id(m)=="Tim Duncan"
+      OPTIONAL MATCH (a)<-[]-(b)
+      RETURN m.player.name AS n1, n.player.name AS n2, a.player.name AS n3 ORDER BY n1, n2, n3 LIMIT 10
+      """
+    Then a ExecutionError should be raised at runtime: Scan vertices or edges need to specify a limit number, or limit number can not push down.
+
+  Scenario: Multi Line Multi Query Parts
+    When executing query:
+      """
+      USE nba;
+      MATCH (m)-[]-(n) WHERE id(m)=="Tim Duncan"
+      WITH n, n.player.name AS n1 ORDER BY n1 LIMIT 10
+      MATCH (n)-[]-(l)
+      RETURN n.player.name AS n1,
+      CASE WHEN l.player.name is not null THEN l.player.name
+      WHEN l.team.name is not null THEN l.team.name ELSE "null" END AS n2 ORDER BY n1, n2 LIMIT 10
+      """
+    Then the result should be, in order:
+      | n1            | n2           |
+      | "Aron Baynes" | "Celtics"    |
+      | "Aron Baynes" | "Pistons"    |
+      | "Aron Baynes" | "Spurs"      |
+      | "Aron Baynes" | "Tim Duncan" |
+      | "Boris Diaw"  | "Hawks"      |
+      | "Boris Diaw"  | "Hornets"    |
+      | "Boris Diaw"  | "Jazz"       |
+      | "Boris Diaw"  | "Spurs"      |
+      | "Boris Diaw"  | "Suns"       |
+      | "Boris Diaw"  | "Tim Duncan" |
+    When executing query:
+      """
+      USE nba;
+      MATCH (m:player{name:"Tim Duncan"})-[:like]-(n)--()
+      WITH  m,count(*) AS lcount
+      MATCH (m)--(n)
+      RETURN count(*) AS scount, lcount
+      """
+    Then the result should be, in order:
+      | scount | lcount |
+      | 19     | 110    |
+    When executing query:
+      """
+      USE nba;
+      MATCH (m:player{name:"Tim Duncan"})-[:like]-(n)--()
+      WITH  m,n
+      MATCH (m)--(n)
+      RETURN count(*) AS scount
+      """
+    Then the result should be, in order:
+      | scount |
+      | 270    |
+    # Below scenario is not suppoted for the execution plan has a scan.
+    When executing query:
+      """
+      USE nba;
+      MATCH (m)-[]-(n) WHERE id(m)=="Tim Duncan"
+      WITH n, n.player.name AS n1 ORDER BY n1 LIMIT 10
+      MATCH (a)-[]-(b)
+      RETURN a.player.name AS n1, b.player.name AS n2 ORDER BY n1, n2 LIMIT 10
+      """
+    Then a ExecutionError should be raised at runtime: Scan vertices or edges need to specify a limit number, or limit number can not push down.
+
+  Scenario: Multi Line Some Erros
+    When executing query:
+      """
+      USE nba;
+      MATCH (m)-[]-(n) WHERE id(m)=="Tim Duncan"
+      WITH n, n.player.name AS n1 ORDER BY n1 LIMIT 10
+      RETURN m
+      """
+    Then a SemanticError should be raised at runtime: Alias used but not defined: `m'
+    When executing query:
+      """
+      USE nba;
+      MATCH (v:player)-[e]-(v:team) RETURN v, e
+      """
+    Then a SemanticError should be raised at runtime: `v': Redefined alias in a single path pattern.
```

Comment:
Add a test case like `yield 1; match ...`

Reply:
just for USE space

Reply:
Test the error